blog/nodejs-corepack.md

```diff
@@ -0,0 +1,96 @@
+# Corepack: the Node.js' manager of package managers
+
+Corepack is the new built-in tool for preparing the required package manager for our projects. We're going to see what comes with this new package for packagers. 📦
+
+## Installation
+
+We're in track to get the new Node.js LTS v16 at the end of October, without a fancy name assigned yet, and we're getting [Corepack](https://github.com/nodejs/corepack) preinstalled in the default Node.js configuration since [v16.9.0](https://github.com/nodejs/node/releases/tag/v16.9.0). 👌
+
+If we want to use this tool with previous versions we just need to install it as a global package:
+
+```bash
+$ npm install -g corepack
+```
+
+🧠 Remember that, until now, Corepack v0.9.0 was only supporting `pnpm`, `npm` and `yarn`.
+
+## Basic usage
+
+We can use the same package manager that we always use in any machine without worring if it's installed or not, using:
```

Comment:
> We can use the same package manager that we always use in any machine without worring if it's installed or not, using:

We can use the same package manager that we've always used in any machine without worrying if it's installed or not, using:

Reply:
```suggestion
We can use the same package manager that we've always use in any machine without worrying if it's installed or not, using:
```